src/Microsoft.Health.Dicom.Core.UnitTests/Features/CustomTag/AddCustomTagServiceTests.cs

```diff
@@ -18,19 +18,19 @@
 
 namespace Microsoft.Health.Dicom.Core.UnitTests.Features.ChangeFeed
 {
-    public class CustomTagServiceTests
+    public class AddCustomTagServiceTests
```

Comment:
wondering why split customTag APIs into 2 services?

Reply:
It's just to keep the services small and easy to understand! I suspect that Add especially might get long once we add support for a job framework!

---
In reply to: [558512075](https://github.com/microsoft/dicom-server/pull/501#discussion_r558512075) [](ancestors = 558512075)

Reply:
There are multiple operations for CustomTag(add/remove/list/get), they might share some functions, putting in same class could make this simpler.
Another concern is if we put AddCustomTag into separated service, should we also put remove/get/list into different classes as well.

Anyway, I'm fine with current approach, if do see there is a need for change, we can do that later.